classes/person.rb

```diff
@@ -0,0 +1,28 @@
+require './nameable'
+
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Please ensure that the `Person` class inherits from `Nameable`. This is a project requirement as you can see below 👇 
![image](https://github.com/sameedmohsin28/OOP-school-library/assets/46839548/7bffd9d7-0034-4d27-853f-489b83364a61)

- [ ] Please replace `require` with `require_relative`. `require` is ideally used when you want to include dependencies, while `require_relative` is ideally suited to when you need to bring in your code from another file in the same project 😎

Reply:
@codepantha Thanks for the explanation